docs/topics_5/web-running-saved-analysis.adoc

```diff
@@ -0,0 +1,16 @@
+
+
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='running_saved_analysis_{context}']
+= Running a saved analysis
+
+You can run the last saved version of a project from the *Analysis results* screen.
+
+.Procedure
+
+. In the web console, click *Analysis results*.
+. Select a *Project*.
```

Comment:
If it's generic, it's "a project". No markup or capitalization. It's only bold if you are referring to an interface element.

Reply:
Select -- it's a list named *Project*. 
x-special/nautilus-clipboard
copy
file:///home/rhoch/Pictures/Analysis%20results%20screen.png

Reply:
In that case, it should be `Select a project from the *Project* list.` IBM, 209.

Reply:
Checked. it's a list named *Project*. 
x-special/nautilus-clipboard
copy
file:///home/rhoch/Pictures/Ananlysis%20results.png